docs/source/notebooks/wps_climdex_temp_pctl.ipynb

```diff
@@ -90,8 +90,8 @@
        "\u001b[0mquail\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mclimdex_temp_pctl\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\u001b[0m\n",
```

Comment:
<p>Just realize these both still use "gsl_" as the prefix for the tempfile</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/quail/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>